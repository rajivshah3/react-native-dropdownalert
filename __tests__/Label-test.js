import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../label';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('renders label with text', () => {
  const wrapper = shallow(<Label text={'Hello World'} />);
  const tree = toJson(wrapper);
  expect(tree).toMatchSnapshot();
});
test('renders label without text', () => {
  const wrapper = shallow(<Label text={''} />);
  const tree = toJson(wrapper);
  expect(tree).toEqual("");
});
