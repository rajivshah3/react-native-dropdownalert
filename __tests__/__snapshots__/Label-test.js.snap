// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders label with text 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  numberOfLines={1}
  style={
    Object {
      "backgroundColor": "transparent",
      "color": "white",
      "fontSize": 16,
      "fontWeight": "normal",
      "textAlign": "left",
    }
  }
>
  Hello World
</Text>
`;
